src/popup.html

```diff
@@ -14,12 +14,15 @@
             <a class="header-bar__menu-text" id="Home">Home</a>
             <a class="header-bar__menu-text" id="Edit">Edit skills</a>
             <a class="header-bar__menu-text" id="Import-Export">Import / Export</a>
+            <a class="header-bar__menu-text" id="Timer">Timer</a>
         </div>
 
         <div class="welcome-message">
             <p>Welcome to Life Gamification!<br>
             To add your first skill go to Edit skills.</p>
+            <p>You can use Timer to track work on skills. 1 minute of work gives 1 exp to a certain skill.</p>
```

Comment:
I think that it isn't welcome message but manual. It should be moved to another section in the future.